Description: Changes found in version 1.51-1
 Changes do to upstream code in version 1.51-1, when the package was
 converted to the 3.0 (quilt) format.
Author: unknown
Forwarded: unknown
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2016-03-07

--- /dev/null
+++ naturaldocs-1.51/naturaldocs
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/perl /usr/share/perl5/naturaldocs/NaturalDocs $@
--- /dev/null
+++ naturaldocs-1.51/naturaldocs.1
@@ -0,0 +1,93 @@
+.\"                                      Hey, EMACS: -*- nroff -*-
+.\" First parameter, NAME, should be all caps
+.\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
+.\" other parameters are allowed: see man(7), man(1)
+.TH NATURALDOCS 1 "May 2007"
+.\" Please adjust this date whenever revising the manpage.
+.\"
+.\" Some roff macros, for reference:
+.\" .nh        disable hyphenation
+.\" .hy        enable hyphenation
+.\" .ad l      left justify
+.\" .ad b      justify to both left and right margins
+.\" .nf        disable filling
+.\" .fi        enable filling
+.\" .br        insert line break
+.\" .sp <n>    insert n+1 empty lines
+.\" for manpage-specific macros, see man(7)
+.SH NAME
+NaturalDocs \- an extensible, multi\-language documentation generator
+.SH SYNOPSIS
+.ad l
+.B naturaldocs
+-i <input (source) directory> [-i <input (source) directory> ...] -o <output format> <output directory> [-o <output format> <output directory> ...] -p <project directory> [options]
+.ad b
+.SH DESCRIPTION
+This manual page documents briefly the .B naturaldocs command. Note that \fBnaturaldocs\fP is a wrapper script that on Debian invokes the real \fBNaturalDocs\fP perl program. So, the real, original name of the program is NaturalDocs but on
+Debian systems you invoke it as naturaldocs.
+.PP
+\fBNatural Docs\fP is is an open-source, extensible, multi-language documentation generator. You document your code in a natural syntax that reads like plain English. Natural Docs then scans your code and builds high-quality HTML documentation from it.
+.SH OPTIONS
+A summary of options, extracted from the help printed by the \fB-h\fP
+switch is included below. 
+For a complete description of how NaturalDocs works, see the text
+files in the /usr/share/doc/naturaldocs directory.
+.sp
+\fIRequired parameters\fR:
+.TP
+.B \-i, \-\-input, \-\-source DIR
+Specifies an input (source) directory. Required.
+.br
+Can be specified multiple times.
+.TP
+.B \-o, \-\-output FMT DIR
+Specifies an output format and directory. Required.
+.br
+Can be specified multiple times, but only once per directory. Possible output formats are HTML and FramedHTML.
+.TP
+.B \-p, \-\-project DIR
+Specifies the project directory. Required.
+.br
+There needs to be a unique project directory for every source directory.
+.sp
+.PP
+\fIOptional parameters\fR:
+.TP
+.B \-s, \-\-style STYLE [STYLE ...]
+Specifies the CSS style when building HTML output. If multiple styles are specified, they will all be included in the order given.
+.TP
+.B \-img, \-\-image DIR
+Specifies an image directory. Can be specified multiple times. Start with \fB*\fP to specify a relative directory, as in -img */images.
+.TP
+.B \-do, \-\-documented\-only
+Specifies only documented code aspects should be included in the output.
+.TP
+.B \-t, \-\-tab\-length LEN
+Specifies the number of spaces tabs should be expanded to. This only needs to be set if you use tabs in example code and text diagrams. Defaults to 4.
+.TP
+.B \-xi, \-\-exclude\-input, \-\-exclude\-source
+Excludes an input (source) directory from the documentation. Automatically done for the project and output directories. Can be specified multiple times.
+.TP
+.B \-nag, \-\-no\-auto\-group
+Turns off auto-grouping completely.
+.TP
+.B \-oft, \-\-only\-file\-titles
+Source files will only use the file name as the title.
+.TP
+.B \-r, \-\-rebuild
+Rebuilds all output and data files from scratch. Does not affect the menu file.
+.TP
+.B \-ro, \-\-rebuild\-output
+Rebuilds all output files from scratch.
+.TP
+.B \-q, \-\-quiet
+Suppresses all non-error output.
+.TP
+.B \-?, \-h, \-\-help
+Displays syntax reference.
+.br
+.SH AUTHOR
+NaturalDocs was written by Greg Valure <gregvalure@naturaldocs.org>.
+.PP
+This manual page was written by Federico Di Gregorio <fog@debian.org>,
+for the Debian project (but may be used by others).
